### DISCUSSION TITLE
ARS Sprint 4 Retrospective notes

### DISCUSSION THREAD

**Discussion by drewcdisc** [cdisc-org/analysis-results-standard#34] [External Discussions]
This PDF came from a Miro session. Feel free to share any comments.

[ARS Sprint 4 Retrospective.pdf](https://github.com/cdisc-org/analysis-results-standard/files/10361806/ARS.Sprint.4.Retrospective.pdf)

**Comment by bhavinbusa** [comment]
This is a sample comment
